### ISSUE TITLE
Lo-res image used

### ISSUE THREAD

**Issue by coda-apps** [cubiq/add-to-homescreen#273]
Hi!

I generated favicons using the "every size" option at http://www.favicomatic.com/, based on an original image sized 164 px X 164px.

But when I see the ATH popup on my phone, the images used seems very low res, and pixelated. it cant be using the full size image. Pls see screenshot below. 

What can be done for this?

Thanks

![screenshot_2017-11-15-15-52-15](https://user-images.githubusercontent.com/13675512/32839920-b33b210a-ca1e-11e7-83e8-1936f1f48347.png)

**Comment by BumbleB2na** [comment]
Hi @coda-apps, I had a very good experience using https://realfavicongenerator.net/. You might want to try that. Otherwise, I could help you if you figure out what image file is being used by [debugging mobile safari](https://appletoolbox.com/2014/05/use-web-inspector-debug-mobile-safari/). The ath popup is a regular html element that can be inspected but, that popup disappears so you'll also want to find whichever ath config parameter that makes it so you have to manually dismiss the popup for it to disappear.

**Comment by coda-apps** [comment]
Thanks @BumbleB2na !
That method with debugging mobile safari is interesting! i looked it and tried (even though my secondary mobile is only a iPhone 4S...) and i on my PC - i dont have the option to see my phone in the Develop menu. I guess its either because its an old phone or because i am on a PC. 
I wonder if there is some way to know the default image that ath uses?

**Comment by BumbleB2na** [comment]
Oh ok. Chrome dev tools mobile simulation lets you choose specific iPhone user agents and sets retina display.

**Comment by coda-apps** [comment]
Thanks. Found that i played around with it with various "devices" . but i cant get the ath popup to appear. do the script is evidently now thinking that its a mobile.  did you manage to get it to work in Chrome?

**Comment by BumbleB2na** [comment]
Yes I had it popping up in Chrome when working with this a few months ago.
Ath needs to see the iPhone user agent and the plugin also remembers if
you've seen the popup already so clear your cache, too.

**Comment by coda-apps** [comment]
cleaned my cache and all, but still cant get it to come up in Chrome with the iphone 5/6 selected

**Comment by coda-apps** [comment]
I managed to see the ath pop up properly on my android phone, in Chrome. 
it seemed to be showing the image version called "apple-touch-icon-57x57.png" - but it was evidently stretching it, therefore it looked blurred.
so why would ath use a smaller image, when larger ones exist?
thanks for all your help!
